ibis/backends/pyspark/compiler.py

```diff
@@ -1772,6 +1772,24 @@ def compile_not_null(t, expr, scope, timecontext, **kwargs):
     return ~F.isnull(col) & ~F.isnan(col)
 
 
+@compiles(ops.DropNa)
+def compile_dropna_table(t, expr, scope, timecontext, **kwargs):
+    op = expr.op()
+    table = t.translate(op.table, scope, timecontext)
+    how = op.how.op().value
+    subset = [col.op().value for col in op.subset] if op.subset else None
+    return table.dropna(how=how, subset=subset)
```

Comment:
We need to be careful here - Spark treats Na and None differently so this might be a behavior difference between Spark/Pandas.

@emilyreff7 Do we have test to validate Na and None values?

Reply:
My tests do include both na_col and none_col examples